docs/getting-started/devenv/README.md

```diff
@@ -1,228 +1,51 @@
 
-1. Node/NPM Development Tools
-2. WordPress Development Site
-3. Code Editor
+A development environment includes the tools you need on your computer to successfully develop for the Block Editor. The three essential components are:
 
-## Quick Start
+1.  Code editor
+2.  Node.js development tools
+3.  Local WordPress environment (site)
 
-Here is a summary of the guide. See each section for additional details and explanations.
+## Choosing a code editor
 
-**1. Install Node development tools**
+A code editor is used to write code, and you can use whichever editor you're most comfortable with. The key is having a way to open, edit, and save text files.
 
-Download and install [Node Version Manager](https://github.com/nvm-sh/nvm) (nvm)
+If you do not already have a preferred code editor, [Visual Studio Code](https://code.visualstudio.com/) (VS Code) is a popular choice for JavaScript development among Core contributors. It works well across the three major platforms (Windows, Linux, and Mac) and is open-source and actively maintained by Microsoft. VS Code also has a vibrant community providing plugins and extensions, including many for WordPress development.
 
-```
-curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
-```
+## Installing Node.js development tools
 
-Quit and restart terminal
-Install Node.js v16.
+Node.js (`node`) is an open-source runtime environment that allows you to execute JavaScript outside of the web browser. While Node.js is not required for all WordPress JavaScript development, it's essential when working with modern JavaScript tools and developing for the Block Editor.
 
-```
-nvm install 16
-```
+Node.js and its accompanying development tools allow you to:
 
-**2. WordPress Development Site**
+-   Install and run WordPress packages needed for Block Editor development, such as `wp-scripts`
+-   Setup local WordPress environments with `wp-env` and `wp-now`
+-   Use the latest ECMAScript features and write code in ESNext
+-   Lint, format, and test JavaScript code
+-   Scaffold custom blocks with the `create-block` package
 
-First download, install, and start [Docker Desktop](https://www.docker.com/products/docker-desktop) following the instructions for your OS.
+The list goes on. While modern JavaScript development can be challenging, WordPress provides several tools, like [`wp-scripts`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/) and [`create-block`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block/), that streamline the process and are made possible by Node.js development tools.
 
--   Install WordPress environment tool
+If you are already familiar with `node`, it's recommended that you use the [LTS](https://nodejs.dev/en/about/releases/) (Long Term Support) version, and you will also need Node Package Manager (`npm`), the `npx` script, and Node Version Manager (`nvm`).
```

Comment:
I would rephrase this sentence to surface the importance of `nvm` and highlight the recommended Node version (LTS)

> A Node version manager tool like [`nvm`](https://github.com/nvm-sh/nvm) is strongly recommended to easily change your Node version when required. The **recommended Node version for Block Development is the [Active LTS](https://nodejs.dev/en/about/releases/) (Long Term Support) version**. Any Node installation includes tools such as the Node Package Manager (`npm`) and the Node Package eXecute (`npx`) that will be required to work with some of the [WP packages](https://github.com/WordPress/gutenberg/tree/trunk/packages).